internal/repository/master_index.go

```diff
@@ -65,11 +66,42 @@ func (mi *MasterIndex) ListPack(id restic.ID) (list []restic.PackedBlob) {
 	return nil
 }
 
+// AddPending adds a given blob to list of pedningBlobs
```

Comment:
"to the list of pending blobs".

Also, this method need not be public.

Reply:
changed it.